zip-0316.rst

```diff
@@ -437,8 +441,12 @@ Requirements for both Unified Addresses and Unified Viewing Keys
   all less than 256 bytes and so could use a one-byte length field,
   encodings for experimental types may be longer.)
 
-* Each Receiver, UFVK, or UIVK SHOULD represent an Address or
-  Viewing Key at the ZIP 32 or BIP 44 Account level.
+* Each Receiver and FVK SHOULD represent an Address or Full Viewing
+  Key at the ZIP 32 or BIP 44 Account level.
+
+* Each IVK SHOULD represent an Incoming Viewing Key at the Account
+  level for Sapling or Orchard, or its non-change child extended public
+  key for Transparent P2PKH.
```

Comment:
I don't think this change makes sense to include, because it's a) no longer requirements that apply to both UAs and UVKs, and b) I think both the original requirement and this change are just duplicating the earlier definitions of FVK and IVK encodings.

However, I think what we _should_ have is a requirement of the following form, which I don't believe is currently stated anywhere else in ZIP 316:
```suggestion
* Within a single UA or UVK, any HD-derived Receiver, FVK, or IVK SHOULD
  represent an Address or Viewing Key for the same account (as used in
  the ZIP 32 or BIP 44 Account level).
```

Reply:
Maybe instead of "any HD-derived Receiver, FVK, or IVK" it should read "all HD-derived Receivers, FVKs, and IVKs"